src/tests/ftest/dfuse/bash.py

```diff
@@ -64,109 +54,91 @@ def run_bashcmd(self, il_lib=None, compatible_mode=False):
-                            f"gcc -c -o {fuse_root_dir}/obj_b.o {fuse_root_dir}/src_b.c",
-                            f"ar -rc {fuse_root_dir}/lib.a {fuse_root_dir}/obj_a.o "
-                            f"{fuse_root_dir}/obj_b.o",
-                            f"objdump -d {fuse_root_dir}/obj_a.o",
-                            f"grep print {fuse_root_dir}/src.c",
-                            "awk '{print $1}' " + fuse_root_dir + '/src.c',
-                            f"base64 {fuse_root_dir}/src.c",
-                            f"md5sum {fuse_root_dir}/src.c",
-                            f"cksum {fuse_root_dir}/src.c",
-                            f"bzip2 -z {fuse_root_dir}/lib.a",
-                            f"chmod u-r {fuse_root_dir}/lib.a.bz2",
-                            "fio --readwrite=randwrite --name=test --size=\"2M\" --directory "\
-                            f"{fuse_root_dir}/ --bs=1M --numjobs=\"4\" --ioengine=psync "\
-                            "--group_reporting --exitall_on_error --continue_on_error=none",
-                            f"curl \"https://www.google.com\" -o {fuse_root_dir}/download.html"]
-                for cmd in commands:
-                    try:
-                        # execute bash cmds
-                        ret_code = general_utils.pcmd(
-                            self.hostlist_clients, env_str + cmd, timeout=30)
-                        if 0 not in ret_code:
-                            error_hosts = NodeSet(
-                                ",".join(
-                                    [str(node_set) for code, node_set in
-                                     list(ret_code.items()) if code != 0]))
-                            raise CommandFailure(
-                                f"Error running '{cmd}' on the following hosts: {error_hosts}")
-                    # report error if any command fails
-                    except CommandFailure as error:
-                        self.log.error("BashCmd Test Failed: %s",
-                                       str(error))
-                        self.fail("Test was expected to pass but "
-                                  "it failed.\n")
-
-                # stop dfuse
-                self.stop_dfuse()
-                # destroy container
-                self.container.destroy()
-            # destroy pool
-            self.pool.destroy()
+        self.add_pool(connect=False)
+        self.add_container(self.pool)
+        mount_dir = f"/tmp/{self.pool.identifier}_daos_dfuse"
+        self.start_dfuse(self.hostlist_clients, self.pool, self.container, mount_dir=mount_dir)
+        if il_lib is not None:
+            # unmount dfuse and mount again with caching disabled
+            self.dfuse.unmount(tries=1)
+            self.dfuse.update_params(disable_caching=True)
+            self.dfuse.update_params(disable_wb_cache=True)
+            self.dfuse.run()
```

Comment:
Again, pre-existing, so not a blocker. But if we only use one pool now, this can be simplified like this. I don't think there was any real need to unmount dfuse and remount with different params. We could just disable caching to begin with.
```suggestion
        if il_lib is None:
            self.start_dfuse(self.hostlist_clients, self.pool, self.container)
        else:
            self.start_dfuse(self.hostlist_clients, self.pool, self.container, disable_caching=True, disable_wb_cache=True)
```